RecoLocalMuon/GEMSegment/test/runGEMSegmentProducer_cfg.py

```diff
@@ -0,0 +1,97 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
this is essentially the same file as this is essentially the same file as RecoLocalMuon/GEMSegment/test/runME0SegmentProducer_cfg.py

please remove this one (or the other one)